lib/xml.js

```diff
@@ -4,10 +4,10 @@ var libxmljs = require('libxmljs');
 
 exports.parse = function (src) {
   var options = {
-    noblanks: true
+    preserveWhitespace: false
   };
 
-  return libxmljs.parseXmlString(src, options);
+  return libxmljs.parseXml(src, options);
```

Comment:
did we measure performance differences? between `parseXmlString` vs `parseXml`

Reply:
parseXmlString deprecated in latest version